homeassistant/components/zwave.py

```diff
@@ -92,6 +93,10 @@
      [COMMAND_CLASS_DOOR_LOCK],
      TYPE_BOOL,
      GENRE_USER),
+    ('rollershutter',
+     [COMMAND_CLASS_SWITCH_MULTILEVEL],
```

Comment:
Ofcourse untested :pray: , neither of us have that combination.
Anyone that do have a dimmer and a rollershutter device that can test?